test/index.js

```diff
@@ -7,8 +7,32 @@ describe('Core', function () {
   var gedcomStream,
       data;
 
+  var expected = {
+    'nested': [
+      { id: 'I1', name: 'INDI', value: '', children: [
+        { name: 'NAME', value: 'Joe Bloggs', children: [] }
+      ] },
+      { id: 'I2', name: 'INDI', value: '', children: [
+        { name: 'NAME', value: 'Jim Bliggs', children: [] },
+        { name: 'ASSO', value: '@I1@', children: [
+          { name: 'RELA', value: 'Godfather', children: [] }
+        ] }
+      ] }
+    ],
+    'single': [
+      { id: 'I1', name: 'INDI', value: '', children: [
+        { name: 'NAME', value: '', children: [] }
+      ]},
+      { id: 'I2', name: 'INDI', value: '', children: [
+      ]}
+    ],
+  };
+
   beforeEach(function () {
     gedcomStream = new GedcomStream();
+    // for some reason the event listeners persist even after recreating the object
+    // which makes the 2nd test case fire the first test comparison
+    gedcomStream.removeAllListeners('end');
```

Comment:
This made the tests pass in master.  I'm guessing something changed in how node caches event handlers such that the event handlers persisted to new instances of the same object, but I couldn't find anything definitive to support that hypothesis.  Without this, it runs the ```assert.deepEqual``` comparison from the first test on the 'end' event from the 2nd test.